php-malware-finder/whitelists/vendors/23642.yar

```diff
@@ -0,0 +1,19 @@
+private rule Vendor23642
+{
+	condition:
+		/* guzzlehttp */
+		hash.sha1(0, filesize) == "565eccf9f43d106028216cddfcba51af675d99cd" or // CachingStream.php
+		hash.sha1(0, filesize) == "b67dcc19adbb175adc875e8d72844f5bce320bca" or // Utils.php
```

Comment:
These two files don't fail the malware scanner for me (and so don't strictly need whitelisting). If they fail the linter it's still worth having them here, I suppose, as they share a whitelist nowadays, but double-check!

Reply:
On my end they do fail with `DodgyStrings` - if I clear the  `23642.yar` file from all the rules